apiservers/engine/backends/network.go

```diff
@@ -114,7 +114,7 @@ func (n *Network) CreateNetwork(name, driver string, ipam apinet.IPAM, options m
 		IPAM:      pools,
 	}
 
-	created, err := PortLayerClient().Scopes.Create(scopes.NewCreateParams().WithConfig(cfg))
+	created, err := PortLayerClient().Scopes.CreateScope(scopes.NewCreateScopeParams().WithConfig(cfg))
```

Comment:
I actually tried Scopes.Create, but swagger doesn't seem to like generating that (it would generate ScopesPost instead for some reason). I think there may be a conflict between Scopes.Create and Container.Create in the swagger code generation.